.changeset/late-shoes-try.md

```diff
@@ -0,0 +1,6 @@
+---
+'@microsoft/atlas-site': minor
+'@microsoft/atlas-css': minor
+---
+
+Adds .gradient-card component, which applies a gradient border, border-width and border-radius to a container.
```

Comment:
Sorry didn't think about it at first, but this is what we normally write in the changeset when adding new components:

```
Adding gradient-card component and relevant documentation.
```

Reply:
Updated